packages/GA4Client/src/common.js

```diff
@@ -1,3 +1,14 @@
+var EVENT_MAX_LENGTH = 40;
```

Comment:
I would add a comment with a link here https://support.google.com/analytics/answer/9267744?hl=en as a source for the max lengths.